exercises/strain/strain.test.ts

```diff
@@ -0,0 +1,76 @@
+import {keep, discard} from './strain'
```

Comment:
Instead of exposing functions, what do you think of exposing a single object?

Reply:
It technically is an object now.

`import * as strain from './strain'`

Then it could be used like so: `strain.keep` and `strain.discard`.

Thoughts on that? Alternatively, I can wrap the functions in another object.